package.json

```
@@ -31,7 +31,7 @@
     "@gnosis.pm/dex-contracts": "^0.2.11",
     "@types/yargs": "^15.0.4",
     "express": "^4.17.1",
-    "express-prom-bundle": "^6.0.0",
+    "express-prom-bundle": "https://github.com/anxolin/express-prom-bundle#add-missing-metric-name-opt",
```

```suggestion
    "express-prom-bundle": "git+https://github.com/anxolin/express-prom-bundle#add-missing-metric-name-opt",
```
this is a more standard approach that npm also understands, not that we use npm, but still...

---

rerun install if you include this, to update `yarn.lock`